### PR TITLE
Fix alignment tests drawing out of bounds, output more alignment images

### PR DESCRIPTION
**Pull request by KyleMaas**
This fixes a crash I encountered where the alignment test images were trying to draw outside of the bounds of the source image.

It also makes the alignment images save out to multiple different files instead of continually overwriting the same one.

### CODE REVIEW

File: src/extract/qr/mod.rs [hunk 1]
```diff
@@ -5,8 +5,11 @@ use crate::util::{Delta, Point};
 
 use image::GrayImage;
 
+// We need to allow unused imports for the width() and height() calls later on, which are part of
+// GenericImageView.
 #[cfg(feature = "debug-images")]
-use image::{DynamicImage, Rgb};
+#[allow(unused_imports)]
```
[the remark this answers is not in the document]
Comment (KyleMaas) [reply src/extract/qr/mod.rs:11]: Try it without it.  Maybe I was doing something wrong, but when I tried it without GenericImageView being imported, I got an error that width() and height() were from a non-imported trait and I would need to specifically import that trait to get those methods.  Which is why I added the import, which made the non-imported trait error go away but added the warning for the unused import.

But I'm fairly new to Rust.  I may have just been messing this up in some other way I don't understand.

[the remark this answers is not in the document]
Comment (piderman314) [reply src/extract/qr/mod.rs:11]: Nah I tried it locally and it works fine without the `GenericImageView` import. Maybe your IDE was messing up? You can check from the command line as well by running `cargo check --features "debug-images"`. The build checks that run when submitting a pull request also do this so you can give it a go :)

**Comment by KyleMaas** [comment]
Worked without it this time without complaining.  Made the change outside of VSCode and just ran the tests on the command line.

**Comment by piderman314** [comment]
Cool, thanks.

**Pull request merged**